CHANGELOG.md

```
@@ -23,6 +23,16 @@ Format follows [Keep a Changelog](https://keepachangelog.com/).
 - PR template and contributor guide now enforce small, single-purpose PR scope and Linear-linked branch/PR conventions
 - MCP/setup, stability, status, and system-overview docs updated to match current command syntax and tool counts
 
+## [1.2.0] — 2026-03-09
+
+### Added
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)